Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -558,23 +526,37 @@ def get_attributes_command(client: Client, args: Dict[str, str], params: Dict[st
     )
 
 
-def fetch_attributes_command(client: Client, params: Dict[str, str]) -> List[Dict]:
+def fetch_attributes_command(client: Client, params: Dict[str, str]):
     """
-    Wrapper for fetching indicators from the feed to the Indicators tab.
+    Fetching indicators from the feed to the Indicators tab.
     Args:
         client: Client object with request
         params: demisto.params()
     Returns: List of indicators.
+
     """
     tlp_color = params.get('tlp_color')
     reputation = params.get('feedReputation')
     tags = argToList(params.get('attribute_tags', ''))
     feed_tags = argToList(params.get("feedTags", []))
     attribute_types = argToList(params.get('attribute_types', ''))
     query = params.get('query', None)
-    indicators = fetch_indicators(client, tags, attribute_types, query, tlp_color,
-                                  params.get('url'), reputation, feed_tags)
-    return indicators
+    params_dict = clean_user_query(query) if query else build_params_dict(tags=tags, attribute_type=attribute_types, limit=2000,
+                                                                          page=1)
```

Comment:
Are we sure we should start at page 1 and not at page 0?

Reply:
According to docs it stating at page 1
https://www.misp-project.org/openapi/#tag/Attributes/operation/restSearchAttributes

Reply:
I also fixed the default value